### ISSUE TITLE
Form Validation Does Not Meet Accessibility Standards

### ISSUE THREAD

**Issue by jeznorth** [bcgov/bc-wallet-mobile#3288]
Colours currently used for validation of form fields do not meet contrast / accessibility standards. 

## Replication Procedure
- In the Setup Steps, navigate to Step 2, and select to set up with another ID.
- Select 'BC driver's license' and continue through the photo steps.
- Once on the form entry interface, don't fill any fields out and hit 'Continue'

## Observed Behaviour
- The validation text and or field outlines are barely visible / legible.

## Expected Behaviour
- Text should meet minimum accessibility requirements for legibility.
- https://www.figma.com/design/LaNyNJGxNMsxlqUCNTWy1X/BCSC-V4---Phase-1?node-id=2911-210&t=OFRLJxb4fRvmWUNv-1

## Developer Notes
- In the 'Account Nickname' section of the Setup Steps, the validation messages were brightened to meet minimum requirements for legibility, but it appears it did not make it's way into other components. We likely should standardize the validation mechanism so we don't have divergent patterns.

<img width="360" height="800" alt="Image" src="https://github.com/user-attachments/assets/2cd96cb7-6434-4042-a18d-a00d14ce0f72" />

**Comment by jleach** [comment]
Confirmed with @hannahgov the design work is done (see [Figma](https://www.figma.com/design/LaNyNJGxNMsxlqUCNTWy1X/BCSC-V4---Phase-1?node-id=2911-210&t=AveCUNWVLdjZxnlo-1)). This should remain in Phase 1.

**Comment by jeznorth** [comment]
@MacDeluca there's some pretty egregious padding on these inputs now. The validation visual is improved though as expected.

<img width="140" height="300" alt="Image" src="https://github.com/user-attachments/assets/9d312d6d-4c76-4be9-b8ea-1bcca8a55d71" />

**Comment by MacDeluca** [comment]
> [@MacDeluca](https://github.com/MacDeluca) there's some pretty egregious padding on these inputs now. The validation visual is improved though as expected.
> 

Im not seeing this on 5305

**Comment by jleach** [comment]
Confirmed accessibility new look on iOS device (new red alert icons).
